language/types_values_variables.md

```diff
@@ -646,6 +978,28 @@ as negative value.
 
 Defaults to `p`
 
+##### Timespan to String
+
+| Format  | Timespan formats
+| ----    | ------------------
+| s       | formats according to the timespan format string '%D-%H:%M:%S' 
+| p       | programmatic representation - a quoted string formatted according to the timespan format string '%D-%H:%M:%S'
```

Comment:
Yes, sounds sensible. If a user want the actual String, they can switch to using the %s format.